packages/modals/README.md

```diff
@@ -0,0 +1,229 @@
+# `@goodhood/modals`
+
+> React map components
+
+## Install
+
+```
+npm i @goodhood/modals
+```
+
+Install peer dependencies
+```
+npm i react // v16.x.x
+npm i prop-types // v15.x.x
+npm i @babel/runtime // v7.x.x
+npm i clsx // v1.x.x
+npm i @goodhood/icons // >= v2.x.x
+npm i @goodhood/components // >= v6.x.x
+npm i nebenan-ui-kit // >= v5.x.x
+npm i nebenan-helpers // >= v5.x.x
+```
+
+Configurate module
+```js
+import { configure } from '@goodhood/modals';
+
+configure({
+  track: // track function which will be called on modal open and close event
+});
+```
+
+## Include:
+
+```js
+import Modal { Alert, configure } from '@goodhood/modals';
+```
+
+# API
+
+## Modal
+```js
+import { ModalProvider } from '@goodhood/modals';
+
+const App = () => (
+  /* Imperative API:
+  - setModal(modalComponent): support legacy API for showing modals via function call
+  - lock(): locks window scroll
+  - unlock(): unlocks window scroll */
+  <ModalProvider ref={ref}>
+    {/* All modals should be placed inside modal provider */}
+  </ModalProvider>
+);
+```
+
+## Modal
+```jsx
```

Comment:
looks like github doesn't have highlighting for jsx, let's switch to `js` instead

Reply:
why? I checked it works.

Reply:
ok